internal/k8s/controllers/pool_controller.go

```diff
@@ -57,9 +57,16 @@ func (r *PoolReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 		return ctrl.Result{}, err
 	}
 
+	var communities metallbv1beta1.CommunityList
+	if err := r.List(ctx, &communities, client.InNamespace(r.Namespace)); err != nil {
```

Comment:
You need to add the watch below (line 99)

Reply:
done